### PR TITLE
UI: Footer Done

### PR DESCRIPTION
**Pull request by rizwan2000rm**
Footer done

### CODE REVIEW

File: src/components/Footer/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Logo } from '../../images/index'
+
+const Footer = () => {
+  return (
```
Comment (ulhas) [inline src/components/Footer/footer.tsx:5]: We can remove this return statement

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import Footer from '../components/Footer/footer'
```
Comment (ulhas) [inline src/pages/index.tsx:2]: Can we make a barrel import file and import it from there?

File: src/components/Footer/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="bg-info-900 text-info-50">
+      <div className="container flex flex-col items-center gap-4 py-10">
+        <Logo height="32" width="130" />
+        <div className="flex gap-8 md:gap-12 justify-center items-center mt-10">
+          <div className="cursor-pointer hover:opacity-80">Link1</div>
```
Comment (ulhas) [inline src/components/Footer/footer.tsx:10]: Can this be extracted to a component?

File: src/components/Footer/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+          <div className="cursor-pointer hover:opacity-80">Link2</div>
+          <div className="cursor-pointer hover:opacity-80">Link3</div>
+          <div className="cursor-pointer hover:opacity-80">Link4</div>
+        </div>
+        <div className="flex gap-4 justify-center items-center mt-10">
+          <div className="bg-white w-6 h-6 rounded-full"></div>
```
Comment (ulhas) [inline src/components/Footer/footer.tsx:16]: This as well. Helps with readability

**Review by ulhas — CHANGES_REQUESTED** [review]

**Comment by raveracker** [comment]
I am closing this as I have made the changes and pushed it in another branch, will raise a new PR for that

**Pull request closed without merging**